pkg/codegen/internal/test/testdata/simple-enum-schema/docs/tree/v1/rubbertree.md

```diff
@@ -239,7 +239,7 @@ The RubberTree resource accepts the following [input]({{< relref "/docs/intro/co
 <a href="#size_go" style="color: inherit; text-decoration: inherit;">Size</a>
 </span>
         <span class="property-indicator"></span>
-        <span class="property-type"><a href="#treesize">Tree<wbr>Size</a></span>
+        <span class="property-type"><a href="#treesize">string</a></span>
```

Comment:
Shouldn't this remain `TreeSize`?

Reply:
That's a good question. For Go--which is the affected language here--I'm pretty sure that the "correct" type is `string`, but that might not be what we want to render for docs.

Reply:
@lukehoban @cnunciato any opinions on this one? My changes currently cause enum-typed properties to be rendered using their underlying type in the Go docs. Might be tricky to fix.